libbeat/tests/system/test_base.py

```diff
@@ -9,22 +11,65 @@
 class Test(TestCase):
     def test_base(self):
         """
-        Checks if all lines are read from the log file.
+        Basic test with exiting Mockbeat normally
         """
 
         self.render_config_template(
             path=os.path.abspath(self.working_dir) + "/log/*"
         )
-        os.mkdir(self.working_dir + "/log/")
 
-        testfile = self.working_dir + "/log/test.log"
-        file = open(testfile, 'w')
+        exit_code = self.run_mockbeat()
+        assert exit_code == 0
 
-        iterations = 80
-        for n in range(0, iterations):
-            file.write("hello world" + str(n))
-            file.write("\n")
 
-        file.close()
+    def test_no_config(self):
+        """
+        Tests starting without a config
+        """
+        exit_code = self.run_mockbeat()
+
+        assert exit_code == 1
+        assert True == self.log_contains("Loading config file error")
+        assert True == self.log_contains("Failed to read")
+
+
+    def test_invalid_config(self):
```

Comment:
I suggest also a test for the `-testconfig` option with a valid configuration file.